rst_files/linear_algebra.rst

```diff
@@ -11,6 +11,8 @@ Linear Algebra
 
 .. contents:: :depth: 2
 
+Co-authors: `Spencer Lyon <https://github.com/sglyon>`__ and `Victoria Gregory <http://github.com/vgregory757>`__.
```

Comment:
@Nosferican  Based on the https://github.com/QuantEcon/lecture-source-jl/issues/52 I think the intention was that we would ensure these were in the contributors, not coauthors of the lectures.  That was the common plan for all lectures with code contribution.  @jstac can verify.

Reply:
Just a reminder to remove this entirely.  See that #52  for details, but basically you do not need to add in coauthors.

Reply:
The version corresponds to the changed version in master. If the credits pass hasn't been done yet, I think is better to do all of those in one pass rather than in each lecture PR.

Reply:
I think the credits pass was done more than 14 days ago (i.e., if you go to the current master, the names were taken out).

Reply:
Seems like I resolved the conflicts in between having it merged. Will try to make a clean up to make sure it is up to date.

Reply:
No problem.